rules/macos/persistence_account_creation_hide_at_logon.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/01/05"
+maturity = "production"
+updated_date = "2020/01/05"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identify attempt to create a local account that will be hidden from the macOS logon window. This may indicate an attempt
+to not attract user attention while persisting via a separate local account.
```

Comment:
```suggestion
Identifies attempts to create a local account that will be hidden from the macOS logon window. This may indicate an attempt to evade user attention while maintaining persistence using a separate local account.
```

Reply:
re-lint